src/components/PriceChart/PriceChart.tsx

```diff
@@ -37,8 +41,25 @@ export const PriceChart: React.FC<PriceChartArgs> = ({
   const loading = useAppSelector(state =>
     selectPriceHistoriesLoadingByAssetTimeframe(state, assetIds, timeframe),
   )
+
   const color = percentChange > 0 ? 'green.500' : 'red.500'
 
+  if (!loading && !data.length)
```

Comment:
I'd usually use a ternary when there are exactly two return paths to make the possible exit points clear.